### PR TITLE
added tests for CVSB-10300

### PR DESCRIPTION
**Pull request by DeepikaSinghD**

### CODE REVIEW

File: src/test/java/testresults/GetTestResults.java [hunk 1]
```diff
@@ -359,102 +359,5 @@ public void testResultsWithTestVersionAutomaticallySetSame() {
         testResultsSteps.statusCodeShouldBe(200);
         testResultsSteps.valueForFieldInPathShouldBe("[0].testVersion","archived");
         testResultsSteps.valueForFieldInPathShouldBe("[0].testHistory[0].testVersion","archived");
-
```
Comment (dpanzaru-deloitte) [inline src/test/java/testresults/GetTestResults.java:362]: Why were these tests deleted?

File: src/test/java/testresults/PutTestResultsCertificateNotGenerated.java [hunk 2]
```diff
@@ -156,7 +156,6 @@ public void testResultsPut() {
         String testNumber = testResultsSteps.getTestNumber();
 
         // verify that the certificate is created in the S3 bucket
-        testResultsSteps.validateCertificateIsNotGenerated(testNumber,randomVin);
-
+        //testResultsSteps.validateCertificateIsNotGenerated(testNumber,randomVin);
```
Comment (dpanzaru-deloitte) [inline src/test/java/testresults/PutTestResultsCertificateNotGenerated.java:159]: Why was this line commented?

**Comment by dpanzaru-deloitte** [comment]
As a good practice, every file should end in a new line. So no matter if it is a json file, a java class or a readme file we should try to end them all with a empty new line.

File: src/test/java/testresults/GetTestResults.java (hunk 1 above)
Comment (DeepikaSinghD) [reply src/test/java/testresults/GetTestResults.java:362]: these tests are moved to PutTestResults class

**Pull request merged**